### ISSUE TITLE
Wrong user name shown in cloud9 ide.

### ISSUE THREAD

**Issue by arshdkhn1** [JanitorTechnology/janitor#278]
When I hover over account picture on https://..../ide.html, `John Doe` shows as username instead of my actual name. I have reported this issue in past but I dont remember whether I opened a Github issue or not. Also it would be nice to show user profile picture in there too.

**Comment by jankeromnes** [comment]
Yes, the name in Cloud9 should probably be set to the user's preferred `Name` in https://janitor.technology/settings (which is what's using the API you linked to).

This does not need to be done in the c9 fork (which we don't keep up-to-date), but can probably be done in this file:

https://github.com/JanitorTechnology/dockerfiles/blob/master/ubuntu-dev/workspace-janitor.js

This is where we configure Cloud9 IDE for Janitor, and as you can see, we already customize the `dashboardUrl`, the `accountUrl`, the Editor's theme and font size, etc. Maybe some property allows customizing the displayed user name?

**Comment by Coder206** [comment]
I found the place where John Doe gets set, see https://github.com/JanitorTechnology/core/blob/15911b3ed65e69efd27d19683a2ac90c69ca4242/settings/standalone.js#L104-L110
